app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -19,6 +23,18 @@ class HomeAssistantApplication : Application(), GraphComponentAccessor {
 
         AndroidThreeTen.init(this)
         graph = Graph(this, 0)
+        // Start the sensor worker if they start the app. The only other place we start this ia Boot BroadcastReceiver
+        SensorWorker.start(this)
+
+        // This will cause the sensor to be updated every time the OS broadcasts that a cable was plugged/unplugged.
+        // This should be nearly instantaneous allowing automations to fire immediately when a phone is plugged
+        // in or unplugged.
+        registerReceiver(
```

Comment:
Ended up moving this into `HomeAssistantApplication` rather than `SensorWorker` feels like it belongs here more.